nomenclature/testing.py

```diff
@@ -12,13 +12,14 @@ def assert_valid_yaml(path: Path):
 
     # iterate over the yaml files in all sub-folders and try loading each
     error = False
-    for file in path.glob("**/*.yaml"):
-        try:
-            with open(file, "r", encoding="utf-8") as stream:
-                yaml.safe_load(stream)
-        except (yaml.scanner.ScannerError, yaml.parser.ParserError) as e:
-            error = True
-            logger.error(f"Error parsing file {e}")
+    for file in path.glob("**/*"):
+        if file.suffix in {".yaml", ".yml"}:
```

Comment:
Suggest to use this more elaborate formulation to avoid double-indenting

```suggestion
    for file in [f for f in path.glob("**/*") if f.suffix in {".yaml", ".yml"}]:
```